src/utils/version.ts

```diff
@@ -4,7 +4,7 @@ export const formatVersion = (version: string | undefined, versionTruncation: 0
   if (version === undefined) return "";
 
   const versionString = trim(version, ".").replace(new RegExp("_", "g"), ".");
-  const versionParts = versionString.split(".");
+  const versionParts = trim(versionString, " ").split(".");
```

Comment:
I think (really not sure) that you can do `trim(version, ". ")` and it will trim both dots and spaces so you wouldn't need two trims.